src/orders/dto/orderDetail.indto.ts

```diff
@@ -0,0 +1,35 @@
+import { ApiProperty } from '@nestjs/swagger';
+import {
+  IsString,
+  IsNotEmpty,
+  MinLength,
+  MaxLength,
+  IsNumber,
+  Min,
+} from 'class-validator';
+
+export default class OrderDetailInDTO {
```

Comment:
You should use the same file pattern for DTOs:  `*.dto.ts`: e.g., `orderDetailIn.dto.ts`.

Reply:
done